doc/classes/VisualServer.xml

```diff
@@ -538,6 +579,7 @@
 			<argument index="1" name="mask" type="int">
 			</argument>
 			<description>
+				The light mask, binary about wich layers this light occluder gets affected by light.
```

Comment:
Change the , to a .

The second part is very unclear. It's describing which light layers are affected by the occluder. Maybe better to just link to [LightOccluder2D] where it's all described in detail.